contrib/platform/src/com/sun/jna/platform/mac/CoreFoundation.java

```diff
@@ -394,7 +451,7 @@ CFMutableDictionaryRef CFDictionaryCreateMutable(CFAllocatorRef alloc, int capac
      *         {@link #CFDictionaryGetValueIfPresent} to distinguish between a value
      *         that is not found, and a {@code null} value.
      */
-    CFTypeRef CFDictionaryGetValue(CFTypeRef theDict, CFStringRef key);
+    Pointer CFDictionaryGetValue(CFTypeRef theDict, PointerType key);
```

Comment:
Why not stay with `CFTypeRef` and `CFStringRef` here?

Reply:
After your earlier comments on the three-arg function I realized that keys are not required to be `CFStringRef`. I even added a testcase (line 201) that uses an `IntByReference` as a key.  While the primary use case for this class is reading data from the IORegistry (where all the keys are strings) it's more properly a generic pointer.

Reply:
Reading the documentation, I agree